Client/Frontend/Home/ActivityStreamPanel.swift

```diff
@@ -606,6 +609,10 @@ enum ASPingEvent: String {
     case Delete = "DELETE"
     case Dismiss = "DISMISS"
     case Share = "SHARE"
+    case NewTab = "NEWTAB"
+    case NewPrivateTab = "NEWPRIVATETAB"
+    case Bookmark = "BOOKMARK"
+    case Remove = "REMOVE"
```

Comment:
I see you added `.Remove` but don't use it anywhere - forgot to add a line somewhere?